mlrun/config.py

```diff
@@ -141,7 +141,7 @@
         "timeout_mode": "enabled",
         # timeout in seconds to wait for background task to be updated / finished by the worker responsible for the task
         "default_timeouts": {
-            "operations": {"migrations": "3600"},
+            "operations": {"migrations": "3600", "load_project": "600"},
```

Comment:
10 minutes for a `load_project` task is veeeeeeery long. I understand you reused the `dask` timeout, but `dask` has its reasons to take that long - here we're spinning a simple pod that loads a project... I wouldn't go higher than 1 minute for this, unless I'm missing something?

Reply:
you right, now its `60`.